### PR TITLE
Multiple language/grammar fixes in i18n

### PR DESCRIPTION
**Pull request by bdmogal**

### CODE REVIEW

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 1]
```diff
@@ -737,23 +737,23 @@ features:
         name-label: "Name"
         name-placeholder: "Namespace name"
         scheduler-queue-label: "Scheduler Queue"
-        scheduler-queue-placeholder: "Specify the YARN queue name to be associated with this namespace"
+        scheduler-queue-placeholder: "Specify the YARN queue name to be used for programs in this namespace"
```
Comment (bdmogal) [inline cdap-ui/app/cdap/text/text-en.yaml:740]: I don't see this being used anywhere.

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 2]
```diff
@@ -737,23 +737,23 @@ features:
...
-        sld-label: "Specify mapping of namespace resources to the underlying existing storage resources."
+        hive-db-name-placeholder: "Hive database for the new namespace"
+        sld-label: "Specify mapping of namespace resources to the existing underlying storage resources."
         ssd-label: "Namespace Mapping"
       Step3:
         keytab-uri-label: "Keytab URI"
         keytab-uri-placeholder: "Location of keytab file associated with the principal"
         principal-label: "Principal"
-        principal-placeholder: "User to be impersonated-as for jobs in the new namespace"
+        principal-placeholder: "User to be impersonated-as for programs in the new namespace"
```
Comment (nitinmotgi) [inline cdap-ui/app/cdap/text/text-en.yaml]: this is not programs. we don't have that. It's application right ?

**Review by nitinmotgi — COMMENTED** [review]
only one comment.

File: cdap-ui/app/cdap/text/text-en.yaml (hunk 2 above)
Comment (bdmogal) [reply cdap-ui/app/cdap/text/text-en.yaml]: the user is used for running programs. This does not specify where it is configured, just where it is used.

**Comment by bdmogal** [comment]
Got verbal LGTM from @nitinmotgi. Merging/

**Pull request merged**